docs/Reboots.md

```diff
@@ -0,0 +1,29 @@
+# What causes reboots?
+
+Under certain conditions, MachineConfig updates will cause the Machine
+Config Daemon to reboot the nodes affected by the change. This document
+aims to identify exactly what MachineConfig changes cause a reboot, with
+the intent that any changes will not be unexpected.
+
+A MachineConfig change will cause a reboot under the following
+conditions:
+
+-   An OS update is being applied.
+-   Linux kernel arguments have changed.
+-   FIPS (Federal Information Processing Standards) configuration has
+    changed.
+-   systemd units have changed.
+-   Linux kernel type is changed (default &lt;&gt; realtime).
+-   RHCOS (Red Hat CoreOS) Extensions is changed.
+
+A reboot will always occur if a change to the filesystem occurs, with
+the exception of the following files:
+
+-   `/etc/kubernetes/kubelet-ca.crt`
+-   `/var/lib/kubelet/config.json`
+-   `/etc/containers/registries.conf`
+
+It should also be noted that if `/etc/containers/registries.conf` changes,
+CRIO will reload, but the node will not reboot.
+
```

Comment:
Probably also worth noting changing SSH keys doesn't cause a reboot

Reply:
Agreed. I'll add that.